src/openforms/contrib/digid_eherkenning/signals.py

```diff
@@ -0,0 +1,19 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from digid_eherkenning.models.digid import DigidConfiguration
+from digid_eherkenning.models.eherkenning import EherkenningConfiguration
+
+from openforms.contrib.digid_eherkenning.utils import (
+    create_digid_eherkenning_csp_settings,
+)
+
+from .constants import CONFIG_TYPES
```

Comment:
why go through `CONFIG_TYPES`, can't you map the extra directives directly to the model?

in `create_digid_eherkenning_csp_settings`
```py
from .constants import ADDITIONAL_CSP_VALUES

additional_values = ADDITIONAL_CSP_VALUES.get(type(config))
...
```
and then you can drop these "magic strings" & `CONFIG_TYPES` mapping.